server/server.go

```diff
@@ -68,6 +68,7 @@ func main() {
 	e.DELETE("/multisig/:address/tx/:id", h.DeleteTransaction)
 	e.POST("/multisig/:address/sign-tx/:id", h.SignTransaction)
 	e.GET("/multisig/:address/txs", h.GetTransactions)
+	e.DELETE("/multisig/:creator-address/:address", h.DeleteMultisigAccount)
```

Comment:
This API allows anyone to delete multisig accounts if they know the creator's address. @anilCSE are we okay with this one.

Reply:
Oh that's bad. We can instead introduce archive mode may be. Also, let's restrict from hitting this API from direct calls.

Reply:
For restricting unauthorized access to delete multisig account api, We can use keplr's signArbitrary API for user verification.
signArbitrary takes chainID, address, and any data string. If user approves that, it is returning a signature.
We can store that signature in localstorage and hash of signature in our database.
For delete multisig account api we can send the signature from localStorage as bearer token in payload and verify the signature with hash stored in DB before we delete account data in our db.
Can we use this?
https://docs.keplr.app/api/#request-signature-for-arbitrary-message

![image](https://github.com/vitwit/resolute/assets/56679184/3f42fb75-79ab-43d9-84cf-9b656955c39c)